routes/api_v1_0.py

```diff
@@ -434,11 +434,11 @@ def get_data_v1_0():
             ) as ds_bearing:
                 bearings = ds_bearing.nc_data.get_dataset_variable("bearing")[
                     data_slice
-                ]
+                ].squeeze(drop=True)
```

Comment:
Move the call to `squeeze` here so we don't have to do a None check when we pass it into the geojson function.